src/components/MenuPanel/MenuPanel.js

```diff
@@ -170,40 +170,71 @@ class MenuPanel extends React.PureComponent {
   }
 }
 
-const AnimatedMenuPanel = ({
-  opened,
-  autoClosing,
-  onCloseMenuPanel,
-  ...props
-}) => {
-  return (
-    <React.Fragment>
-      <Spring
-        from={{ progress: 0 }}
-        to={{ progress: Number(opened) }}
-        config={springs.lazy}
-        immediate={!autoClosing}
-        native
-      >
-        {({ progress }) => (
-          <Wrap
-            style={{
-              transform: progress.interpolate(
-                v =>
-                  `translate3d(
-                    calc((${-100 * (1 - v)}%) - (${SHADOW_WIDTH *
-                    (1 - v)}px)), 0, 0)`
-              ),
-              opacity: progress.interpolate(v => (v > 0 ? 1 : 0)),
-            }}
-          >
-            <MenuPanel {...props} />
-          </Wrap>
-        )}
-      </Spring>
-      <Overlay opened={opened} onClick={onCloseMenuPanel} />
-    </React.Fragment>
-  )
+class AnimatedMenuPanel extends React.Component {
+  state = {
+    animate: false,
+  }
```

Comment:
Would you add `_timer = null` to make it explicit there is a class prop being used?